src/index.js

```diff
@@ -0,0 +1,9 @@
+import { validate } from "./validate.js";
+
+const [content] = process.argv.slice(2);
+
+const run = async () => {
+  console.log((await validate(content)).length);
```

Comment:
If this is 0, would the presence of the log trigger an error in the bash line above?

```
flag=$(node index.js "$content")
```

Reply:
Oh maybe I need to be more explicit 🤔 

```suggestion
  console.log((await validate(content)).length > 0);
```